controllers/informationCenter/SubmissionInformationCenterHandler.inc.php

```diff
@@ -19,6 +19,43 @@
 
 class SubmissionInformationCenterHandler extends InformationCenterHandler {
 
+	/** @var boolean Is the current user assigned to an editorial role for this submission */
+	var $_isCurrentUserAssignedEditor;
+
+	/**
+	 * @copydoc PKPHandler::authorize()
+	 */
+	function authorize($request, &$args, $roleAssignments) {
+		$success = parent::authorize($request, $args, $roleAssignments);
+
+		// Prevent users from accessing history unless they are assigned to an
+		// appropriate role in this submission
```

Comment:
So the other managers or admin, that are not assigned to this submission will not be able to see the history, correct? They can see everything so why not history?